pkg/db-repo/change_history_manager.go

```diff
@@ -224,7 +224,7 @@ func (c *changeHistoryManager) execute() error {
 		if err != nil {
 			c.logger.WithFields(mon.Fields{
 				"sql": statement,
-			}).Error(err, "")
+			}).Error(": %w", err)
```

Comment:
This looks like it could need a proper error message

Reply:
added the message from the error return..